packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -4998,6 +4998,86 @@ void main() {
     );
   });
 
+  testWidgets('Receive repeat [TextEditingValue] will not trigger the request keyboard', (WidgetTester tester) async {
```

Comment:
"Repeatedly receiving [TextEditingValue] will not trigger a keyboard request"

Reply:
Done.
Thanks for review quickly :)